src/main/java/codecool/view/ConsoleColors.java

```diff
@@ -0,0 +1,76 @@
+package codecool.view;
+
+public class ConsoleColors {
+    // Reset
+    public static final String RESET = "\033[0m";  // Text Reset
+
+    // Regular Colors
```

Comment:
YAGNI ?

Reply:
I don't understand? You mean the entire ConsoleColors class or String RESET? 
If I remind example well: if you won't reset settings will be provided to rest of the following strings / texts.

Reply:
I mean all colors listed below line 7 - you use barely any.